.github/workflows/stalebot.yml

```
@@ -0,0 +1,24 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 9 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "Hi, It's been a while since we last heard from you. We are marking this issue as stale due to inactivity. Please, provide the requested feedback, or we will have to close the issue in the next 7 days."
```

```suggestion
        stale-issue-message: "Hi, it's been a while since we last heard from you. We are marking this issue as stale due to inactivity. Please provide the requested feedback or the issue will be closed after next 7 days."
```

I know we have "Hi, It's..." in our templates but I think it's not correct that way? Unless, there are line breaks and it's like e.g. email when you write:

```
Hello Jacek,

It's been...
```

:thinking:

---

Not 100% sure about the grammar but it seems there were too many `,` there :thinking: Can you also update pr message accordingly?

---

I would just remove "Hi" since it should be a continuation of the discussion, so a user should be already greeted by us.